schema/namespace_test.go

```diff
@@ -7,14 +7,14 @@ import (
 )
 
 func TestExpand(t *testing.T) {
-
+http: //purl.org/iot/vocab/thingful
```

Comment:
this looks like a mistake, but you've revealed a larger mistake; wercker is not running the tests in the `schema` package.